pkg/wallet/wallet.go

```diff
@@ -116,30 +116,24 @@ func FromYAML(raw []byte) (w Wallet, err error) {
 	return
 }
 
-func GetWalletDirectory() string {
-	walletDirectory := os.Getenv("HOME") + "/.config/thyra/"
-	if _, err := os.Stat(walletDirectory); os.IsNotExist(err) {
-		if os.MkdirAll(walletDirectory, WalletDirectoryPermission) != nil {
-			walletDirectory = ""
-		}
-	}
-
-	return walletDirectory
-}
-
 func LoadAll() (wallets []Wallet, e error) {
 	wallets = []Wallet{}
 
-	files, err := os.ReadDir(GetWalletDirectory())
+	configDir, err := config.GetConfigDir()
 	if err != nil {
-		return nil, fmt.Errorf("reading wallet directory '%s': %w", GetWalletDirectory(), err)
+		return nil, fmt.Errorf("reading config directory '%s': %w", configDir, err)
+	}
+
+	files, err := os.ReadDir(configDir)
+	if err != nil {
+		return nil, fmt.Errorf("reading wallet directory '%s': %w", configDir, err)
 	}
 
 	for _, f := range files {
 		fileName := f.Name()
 
 		if strings.HasPrefix(fileName, "wallet_") && strings.HasSuffix(fileName, ".json") {
-			bytesInput, err := os.ReadFile(GetWalletDirectory() + fileName)
+			bytesInput, err := os.ReadFile(configDir + fileName)
```

Comment:
I think there is a bug here, I got `reading file 'wallet_nathan.json': open /Users/nathanmassa/.config/thyrawallet_nathan.json: no such file or directory`.

We should always use path.join